src/Optimism.sol

```diff
@@ -3,6 +3,57 @@ pragma solidity >=0.8.0;
 
 library Optimism {
 
+    /******************************************************************************************************************/
+    /*** Token Addresses                                                                                            ***/
+    /******************************************************************************************************************/
+
+    address internal constant SUSDS = 0xb5B2dc7fd34C249F4be7fB1fCea07950784229e0;
+    address internal constant USDC  = 0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85;
+    address internal constant USDS  = 0x4F13a96EC5C4Cf34e442b46Bbd98a0791F20edC3;
+
+    /******************************************************************************************************************/
+    /*** Bridging Addresses                                                                                         ***/
+    /******************************************************************************************************************/
+
+    address internal constant CCTP_TOKEN_MESSENGER = 0x2B4069517957735bE00ceE0fadAE88a26365528f;
```

Comment:
Verified this address is correct from https://developers.circle.com/stablecoins/evm-smart-contracts